packages/cactus-plugin-ledger-connector-iroha/src/test/typescript/integration/run-transaction-endpoint-v1.test.ts

```diff
@@ -0,0 +1,786 @@
+} from "@hyperledger/cactus-test-tooling";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+import { PluginImportType } from "@hyperledger/cactus-core-api";
+
+import {
+  IListenOptions,
+  LogLevelDesc,
+  Servers,
+} from "@hyperledger/cactus-common";
+
+import {
+  PluginLedgerConnectorIroha,
+  DefaultApi as IrohaApi,
+  //RunTransactionRequest,
+  PluginFactoryLedgerConnector,
+  RunTransactionRequest,
+} from "../../../main/typescript/public-api";
+
+import { Configuration } from "@hyperledger/cactus-core-api";
+
+import {
+  IrohaCommand,
+  IrohaQuery,
+} from "../../../main/typescript/generated/openapi/typescript-axios";
+
+const testCase = "runs tx on an Iroha v1.2.0 ledger";
+const logLevel: LogLevelDesc = "INFO";
+
+test.onFailure(async () => {
+  await Containers.logDiagnostics({ logLevel });
+});
+
+test("BEFORE " + testCase, async (t: Test) => {
+  const pruning = pruneDockerAllIfGithubAction({ logLevel });
+  await t.doesNotReject(pruning, "Pruning didn't throw OK");
+  t.end();
+});
+
+test(testCase, async (t: Test) => {
+  const postgres = new PostgresTestContainer({ logLevel });
+
+  test.onFinish(async () => {
+    await postgres.stop();
+  });
+
+  await postgres.start();
+  const postgresPort = await postgres.getPostgresPort();
+  const postgresHost = await internalIpV4();
+  if (!postgresHost) {
+    throw new Error("Could not determine the internal IPV4 address.");
+  }
+  const iroha = new IrohaTestLedger({
+    postgresHost,
+    postgresPort,
+    logLevel,
+  });
+
+  test.onFinish(async () => {
+    await iroha.stop();
+  });
+  await iroha.start();
+
+  const rpcToriiPortHost = await iroha.getRpcToriiPortHost();
+  const factory = new PluginFactoryLedgerConnector({
+    pluginImportType: PluginImportType.Local,
+  });
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const connector: PluginLedgerConnectorIroha = await factory.create({
+    rpcToriiPortHost,
+    instanceId: uuidv4(),
+    pluginRegistry: new PluginRegistry(),
+  });
+
+  const expressApp = express();
+  expressApp.use(bodyParser.json({ limit: "250mb" }));
+  const server = http.createServer(expressApp);
+  const listenOptions: IListenOptions = {
+    hostname: "0.0.0.0",
+    port: 0,
+    server,
+  };
+  const addressInfo = (await Servers.listen(listenOptions)) as AddressInfo;
+  test.onFinish(async () => await Servers.shutdown(server));
+  const { address, port } = addressInfo;
+  const apiHost = `http://${address}:${port}`;
+  const apiConfig = new Configuration({ basePath: apiHost });
+  const apiClient = new IrohaApi(apiConfig);
+
+  await connector.getOrCreateWebServices();
+  await connector.registerWebServices(expressApp);
+
+  let firstTxHash;
+  const adminAccount = iroha.getGenesisAdminAccount();
+  {
+    const req = {
+      commandName: IrohaCommand.CreateAccount,
+      params: [
+        "user1",
+        "test",
+        "fe31d7bc2dbe695b1cd5c706dde902d5fb6e6ce4b60e08418c842fdef5805230", //node4.pub
```

Comment:
I suggest to have const instead of comment.
And here You are creating account, so the name `node` rather is not good, of course keys for nodes and accounts has the same pattern, but the name can be misunderstanding:)

Reply:
Thank you for pointing out. I removed the magical string by generating a random public key. (used const)